community/docker/APKBUILD

```diff
@@ -127,12 +136,13 @@ package() {
 	install -Dm755 "$_containerd_builddir"/bin/ctr \
 		"$pkgdir"/usr/bin/docker-containerd-ctr || return 1
 
-	install -Dm755 bundles/$ver/dynbinary/dockerinit-$ver \
-		"$pkgdir"/usr/lib/docker/dockerinit
+	install -Dm755 "$_tini_builddir"/tini-static \
+		"$pkgdir"/usr/bin/docker-init || return 1
+
 	install -Dm755 contrib/init/openrc/docker.initd \
-		"$pkgdir"/etc/init.d/docker
+		"$pkgdir"/etc/init.d/docker || return 1
```

Comment:
we should keep the `|| return 1` til after 3.6-stable is branched. (when v3.6.0 is released)

http://lists.alpinelinux.org/alpine-devel/5608.html